src/modules/SSHAuthorizedKeys.rb

```diff
@@ -39,6 +45,27 @@ def read_keys(home)
     # @see Yast::Users::SSHAuthorizedKeyring#write_keys
     def write_keys(home)
       keyring.write_keys(home)
+    rescue Users::SSHAuthorizedKeyring::HomeDoesNotExist
+      log.warn("Home directory '#{home}' does not exist")
+      Report.Warning(
+        # TRANSLATORS: '%s' is a directory path
+        _(format("Home directory '%s' does not exist\n" \
```

Comment:
It's the other way round, first translate then format. Otherwise the translation won't be found. I.e. use `format(_("..."), foo)`.